test/startservers.py

```diff
@@ -15,31 +18,34 @@ def do_GET(self):
             self.end_headers()
             self.wfile.write("Do What Ye Will (An it Harm None).\n")
     def run(self):
-        BaseHTTPServer.HTTPServer(("localhost", 4001), self.ToSHandler).serve_forever()
+        try:
+            BaseHTTPServer.HTTPServer(("localhost", 4001), self.ToSHandler).serve_forever()
+        except Exception as e:
+            print "Problem starting ToSServer: %s" % e
+            exit(1)
```

Comment:
looks like `exit` is more meant for the interpreter, not proper programs? http://stackoverflow.com/a/6501134/329700